src/sentry/api/serializers/models/organization_member/base.py

```diff
@@ -28,11 +29,13 @@ def get_attrs(
         # Preload to avoid fetching each user individually
         prefetch_related_objects(item_list, "user", "inviter")
         users_set = {
-            organization_member.user
+            organization_member.user_id
             for organization_member in item_list
             if organization_member.user_id
         }
-        users_by_id = get_serialized_users_by_id(users_set, user)
+        users_by_id: Mapping[int, Any] = {
+            u["id"]: u for u in user_service.serialize_users(user_ids=list(users_set))
```

Comment:
Adding the user serializer as an RPC endpoint has been a god send -- we do a lot of user serialization from region endpoints.